<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>One Key Create</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Tel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Admin ID card</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditAdminId"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Legal Person Name</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Document Date</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditCorpId"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditAdminName"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEditDept"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Admin Department</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Corporate ID Card</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditOrgName"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Admin Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditOrgPhone"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEditPost"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditCorpName"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Organization</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Admin Post</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditOrgName</tabstop>
  <tabstop>lineEditOrgPhone</tabstop>
  <tabstop>lineEditCorpName</tabstop>
  <tabstop>lineEditCorpId</tabstop>
  <tabstop>lineEditAdminName</tabstop>
  <tabstop>lineEditAdminId</tabstop>
  <tabstop>lineEditDept</tabstop>
  <tabstop>lineEditPost</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
